packages/main/src/components/settings-tab/SettingsTab.jsx

```diff
@@ -68,48 +74,44 @@ const SettingsTab = () => {
 
       <div className={styles.settingsTab} id={styles.settingtab}>
         <div className={styles.settingleft} id={styles.settingleft}>
-          Sign out all other sessions <br />
-          <span>
-            Lost your phone? Left yourself logged in on a public computer? Need
-            a way to sign out from everywhere except your current browser? Then
-            this is for you
-          </span>
+          {t("sign_SettingsTab")} <br />
+          <span>{t("lost_SettingsTab")}</span>
         </div>
         <div className={styles.settingsright}>
           <Link to="/session-signout">
             <button className={styles.signout}>
-              Sign out of all othe sessions
+              {t("signout_SettingsTab")}
             </button>
           </Link>
         </div>
       </div>
 
       <div className={styles.settingsTab} id={styles.settingtab}>
         <div className={styles.settingleft} id={styles.settingleft}>
-          Deactivate Account <br />
-          <span>
-            If you no longer need your account for the Zurichat, you can
-            deactivate your account. Any other Zurichat workspaces you belong to
-            will not be affected.
-          </span>
+          {t("acc_SettingsTab")} <br />
+          <span>{t("zuri_SettingsTab")}</span>
```

Comment:
Ensure your keys are following one standard. They should all the small letters with _ in between words. I'm seeing zuri_SettingsTab, another one Save_Lang. Make sure they're all small letters

Also, why Zuri chat page was scrapped. Remove any change that's related to that page